Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7.11
+FROM python:2.7
```

Comment:
@mjstealey Just curious, changing it to python:2.7 without sub-version number would pull which sub-version in version 2.7 line?